test/contracts/Wallet.cpp

```diff
@@ -559,24 +559,24 @@ BOOST_AUTO_TEST_CASE(multisig_value_transfer)
 	BOOST_REQUIRE(callContractFunction("addOwner(address)", h256(account(3), h256::AlignRight)) == encodeArgs());
 	// 4 owners, set required to 3
 	BOOST_REQUIRE(callContractFunction("changeRequirement(uint256)", u256(3)) == encodeArgs());
-	// check that balance is and stays zero at destination address
-	BOOST_CHECK_EQUAL(balanceAt(Address(0x05)), 0);
+	Address destination = account(5);
+	auto destinationBalance = balanceAt(destination);
```

Comment:
Which one is better? This option or sending off all the balance first (see below).

Reply:
Oh never mind, only the first account ever created gets a balance in RPCSession.

Reply:
Probably better creating a helper `emptyAccount` which forces creation of a new one.